### PR TITLE
Feature/add getting started info to readme

### PR DESCRIPTION
**Pull request by ashulmanWeWork**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
...
+### Common Calls and Examples
+There are two main ways to use the client:
+1. Set the namespace
+2. Supply the namespace for calls that require it
+
+#### Instantiating the client
+```mc = Client()```
+
+#### Set a Namespace for the client to use
+```mc.set_namespace(existing_namespace)```
```
Comment (roaraya8) [inline README.md]: I don't think this is supported anymore. You either set the namespace in the constructor or you pass it in every call.
There is no such set_namespace()

File: README.md [hunk 2]
```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
...
+#### Create a Namespace
+``` 
+mc.create_namespace(namespace_name, 
+    "owner name",
+    "description of namespace")
+   ```
+
+#### Create a Job
+```
+mc.create_job(job_name=job_name,
```
Comment (sshah-wework) [inline README.md]: this example uses all named parameters, but the call to `create_namespace` does not. can we standardize in the example?

File: README.md [hunk 3]
```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
...
+
+#### Instantiating the client
+```mc = Client()```
+
+#### Set a Namespace for the client to use
+```mc.set_namespace(existing_namespace)```
+
+#### Create a Namespace
+``` 
+mc.create_namespace(namespace_name,
```
Comment (sshah-wework) [inline README.md]: `namespace_name` isn't defined in your example -- I think putting an example string into the call is fine, just like you're doing with owner name.

File: README.md [hunk 4]
```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
...
+    location=job_location,
+    description=description,
+    input_dataset_urns=input_dataset_urns,
+    output_dataset_urns=output_dataset_urns,
+    namespace_name=namespace_name)
+```
+
+#### Create a JobRun
+```
+job_run_info = marquez_client.create_job_run(
```
Comment (sshah-wework) [inline README.md]: the client object is called `marquez_client` here, but everywhere else it is `mc`

File: README.md (hunk 1 above)
Comment (ashulmanWeWork) [reply README.md]: Good call. Adding more info about the ways to use the env vars or constructor to set.

File: README.md [hunk 5]
```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
...
+    job_name=job_name,
+    namespace_name=namespace_name)
+```
+
+```
+{
+    "runId": "870492da-ecfb-4be0-91b9-9a89ddd3db90",
+    "runState": "COMPLETED",
+    "runArgs": "{\"email\": \"data@wework.com\"}"
+}
```
Comment (sshah-wework) [inline README.md:136]: might be good to indicate that this is the data in `job_info` ^^

File: README.md (hunk 4 above)
Comment (ashulmanWeWork) [reply README.md]: Good catch.

File: README.md (hunk 2 above)
Comment (ashulmanWeWork) [reply README.md]: No problem. Adjusted the code to use a more real-world example with inline values.

File: README.md (hunk 3 above)
Comment (ashulmanWeWork) [reply README.md]: Makes sense. Will remove undefined vars.

**Review by roaraya8 — APPROVED** [review]
Looks good!

File: README.md (hunk 2 above)
Comment (sshah-wework) [reply README.md]: hm, not sure this should've been marked resolved. this was not addressed:


```
mc.create_namespace("accounting_team", 
    "Gabriel G. Marquez",
    "Jobs and data sets related to accounting.")
```

The above are not named parameters, but the next example uses them:

```
mc.create_job(job_name="quarterly_summary",
    location="accounting_summary.py",
    description="generates the quarterly accounting report",
    input_dataset_urns=["urn:dataset:accounting_db:monthly_rollups_q2_2019"],
    output_dataset_urns=["urn:dataset:accounting_db:quarterly_reports_q2_2019"],
    namespace_name="accounting_team")
```

**Pull request merged**